src/propgrid/props.cpp

```diff
@@ -2199,7 +2199,7 @@ bool wxFileProperty::DisplayEditorDialog(wxPropertyGrid* pg, wxVariant& value)
         m_dlgTitle.empty() ? _("Choose a file") : m_dlgTitle,
         m_initialPath.empty() ? path : m_initialPath,
         file,
-        m_wildcard.empty() ? wxALL_FILES : m_wildcard,
+        m_wildcard.empty() ? _(wxALL_FILES) : m_wildcard,
```

Comment:
If I'm not mistaken, this would be the only place in wx source code where `_()` is called with something else than a string literal as the argument. Elsewhere the longer name `wxGetTranslation` is used when the argument is e.g. a variable. This is purely about style, it doesn't change anything per se.

Perhaps the longer name serves as a reminder that the actual string literal needs to be defined somewhere else as well, because `xgettext` presumably doesn't expand macros.

Reply:
Thanks for the comment, I think you are correct.

OTOH, I followed what Vadim suggested wrote in the ticket, so I let him decide.

Reply:
I used `_()` just for brevity, to be honest, I guess `wxGetTranslation()` could be indeed better, but I don't feel strongly enough about this to change it, so I'll just merge it as is.